padguidedb/padguidedb.py

```
@@ -175,8 +198,12 @@ async def fulletl(self, ctx):
 
         self.full_etl_running = True
         await ctx.send(inline('Running full ETL pipeline: this could take a while'))
-        await running_load
-        self.full_etl_running = False
+        try:
+            await running_load
+        except:
```

Just FYI, you don't need to except: pass to use finally.